build_ovtf.py

```diff
@@ -126,16 +103,11 @@ def main():
         "the OpenVINO integration with TensorFlow Library without OpenVINO libraries",
         action="store_true")
 
-    parser.add_argument(
```

Comment:
Why is this removed? The C++ API is enabled by default now?

Reply:
This option was not having any impact after removing build_tf_from_source option. 
C++ APIs are built if TF source location is provided otherwise they are not built.